src/index.js

```diff
@@ -83,6 +83,8 @@ document.addEventListener('DOMContentLoaded', () => {
     setSubtitle(getEntryHead())
     setupAudioOnPageLoad()
     setupParadigmSizeToggleButton()
+  } else if (route === '/query-help' || route == '/admin/fst-tool') {
```

Comment:
Honestly, the router needs to be fixed here in a future PR :/

Reply:
Yeah, I just added this because Cypress was correctly complaining about the exception showing up in the JS console whenever these pages were being visited.